templates/conf.d/nginx.conf.erb

```diff
@@ -53,8 +53,8 @@ events {
 }
 
 http {
-<% if @http_raw_prepend && Array(@http_raw_prepend).size > 0 -%>
-    <%- Array(@http_raw_prepend).each do |line| -%>
+<% if @http_raw_prepend -%>
```

Comment:
I think this is not the line you wanted to change. raw prepend can be both a string and an array so here it is needed.

Reply:
Right. Thanks. Fixed.